### ISSUE TITLE
Is there a way to remove the XML declaration for the xml builder?

### ISSUE THREAD

**Issue by hsheikhali** [Leonidas-from-XIV/node-xml2js#600]
Just wondering if there was a way to remove

`<?xml version="1.0" encoding="UTF-8" standalone="yes"?>`

after running the xml2js.Builder().buildObject()

Reason I am asking is because I'm trying to work with SOAP.. and I don't need extra declarations for building my soap request.

**Comment by wparad** [comment]
Yes, pass in this property to the builder `new xml2js.Builder({ headless: true }).buildObject(json);`
